examples/db-templates/README.md

```diff
@@ -38,33 +38,29 @@ Replace `/path/to/template.json` with an appropriate path, that can be either a
 local path or an URL. Example:
 
     $ oc new-app https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json
-    --> Deploying template mongodb-ephemeral for "https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json"
-     With parameters:
-      DATABASE_SERVICE_NAME=mongodb
-      MONGODB_USER=userJNX # generated
-      MONGODB_PASSWORD=tnEDilMVrgjp5AI2 # generated
-      MONGODB_DATABASE=sampledb
-      MONGODB_ADMIN_PASSWORD=8bYEs8OlNYhVyMBs # generated
-    --> Creating resources ...
-    Service "mongodb" created
-    DeploymentConfig "mongodb" created
-    --> Success
-    Run 'oc status' to view your app.
 
 The parameters listed in the output above can be tweaked by specifying values in
 the command line with the `-p` option:
 
     $ oc new-app examples/db-templates/mongodb-ephemeral-template.json -p DATABASE_SERVICE_NAME=mydb -p MONGODB_USER=default
     --> Deploying template mongodb-ephemeral for "examples/db-templates/mongodb-ephemeral-template.json"
-         With parameters:
-          DATABASE_SERVICE_NAME=mydb
-          MONGODB_USER=default
-          MONGODB_PASSWORD=RPvMbWlQFOevSowQ # generated
-          MONGODB_DATABASE=sampledb
-          MONGODB_ADMIN_PASSWORD=K7tIjDxDHHYCvFrJ # generated
+
+        mongodb-ephemeral
+        ---------
+        MongoDB database service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
+
+        * With parameters:
+          * Memory Limit=512Mi
+          * Namespace=openshift
+          * Database Service Name=mydb
+          * MongoDB User=default
+          * MongoDB Password=0oaNRcbyq4Ed6cNh # generated
+          * MongoDB Database Name=sampledb
+          * MongoDB Admin Password=Drh6VO8hgVsVgolK # generated
+
```

Comment:
@guangxuli could you remove the output of the command here too? That should be all, thanks!